packages/cozy-client/src/hooks/useQuery.js

```diff
@@ -51,7 +51,7 @@ const useQuery = (queryDefinition, options) => {
 
   const client = useClient()
   const queryState = useSelector(() => {
-    if (options.singleDocData === undefined && queryDefinition.id) {
+    if (options.singleDocData === undefined && definition?.id) {
```

Comment:
Good catch!

Reply:
Thanks to typing, yet another example, even with JSDoc